build.gradle

```diff
@@ -156,7 +158,11 @@ jar {
       include 'FitNesseRoot/FrontPage/**/properties.xml'
       include 'FitNesseRoot/PageFooter/**/content.txt'
       include 'FitNesseRoot/PageFooter/**/properties.xml'
+      include 'FitNesseRoot/PlugIns/**/properties.xml'
+      include 'FitNesseRoot/PlugIns/**/content.txt'
       include 'FitNesseRoot/PageHeader/**/content.txt'
+      include 'FitNesseRoot/Plugins/**/properties.xml'
+      include 'FitNesseRoot/Plugins/**/content.txt'
```

Comment:
Why is FitNesseRoot/Plug[iI]ns added twice?

Reply:
What exactly is your question? 
 * I first added it with the wrong casing (lowercase 'i'), and then I corrected that. 
 * Like all others (already present pages) I needed to add a line for both the properties and the content.